src/analysis/ControlFlowGraph.cpp

```diff
@@ -120,22 +121,16 @@ bool ControlFlowLoop::includes(const ControlFlowLoop& other) const
     if(*this == other)
         return false;
 
-    auto head = std::find_if(
-        this->begin(), this->end(), [&](const CFGNode* node) { return node->key == (*other.begin())->key; });
-    if(head == this->end())
+    for(auto otherItr : other)
     {
-        return false;
-    }
-
-    auto thisItr = head;
-    auto otherItr = other.begin();
-    while(thisItr != this->end() && otherItr != other.end() && (*thisItr)->key == (*otherItr)->key)
-    {
-        ++thisItr;
-        ++otherItr;
+        auto thisItr = std::find_if(begin(), end(), [&](const CFGNode* node) { return node->key == otherItr->key; });
```

Comment:
What does this change change in behavior?

Reply:
At currently code, there are some cases which is not detected correctly.

For example, there are following two loops.
![badcase](https://user-images.githubusercontent.com/3138900/46255776-85b6d080-c4dc-11e8-93c8-3f8fdfc6367b.png)
Loop A: a -> b -> c -> d
Loop C: c -> d -> b
Loop A should includes Loop C. But current program returns false.

So I fixed to simply based on definition of "includes".